samples/highcharts/accessibility/internationalization/demo.js

```diff
@@ -0,0 +1,140 @@
+const languages = {
+    no: {
+        lang: {
+            decimalPoint: ',',
+            thousandsSep: '.',
+            loading: 'Later­...',
+            months: ['Januar', 'Februar', 'Mars', 'April', 'Mai', 'Juni', 'Juli', 'August', 'September', 'Oktober', 'November', 'Desember'],
+            shortMonths: ['Jan', 'Feb', 'Mar', 'Apr', 'Mai', 'Jun', 'Jul', 'Aug', 'Sep', 'Okt', 'Nov', 'Des'],
+            weekdays: ['Mandag', 'Tirsdag', 'Onsdag', 'Torsdag', 'Fredag', 'Lørdag', 'Søndag­'],
+            exportButtonTitle: 'Eksporter',
+            printButtonTitle: 'Print',
+            printChart: 'Skriv ut diagram',
+            viewFullscreen: 'Se i fullskjerm',
+            rangeSelectorFrom: 'Fra',
+            rangeSelectorTo: 'Til',
+            rangeSelectorZoom: 'Zoom',
+            downloadPNG: 'Last ned som PNG-bilde',
+            downloadJPEG: 'Last ned som JPEG-bilde',
+            downloadPDF: 'Last ned som PDF-dokument',
+            downloadSVG: 'Last ned som SVG-bilde',
+            title: 'AAPL Aksjekurs'
+        }
+    },
+    de: {
+        lang: {
+            decimalPoint: ',',
+            thousandsSep: '.',
+            loading: 'Daten werden geladen...',
+            months: ['Januar', 'Februar', 'März', 'April', 'Mai', 'Juni', 'Juli', 'August', 'September', 'Oktober', 'November', 'Dezember'],
+            weekdays: ['Sonntag', 'Montag', 'Dienstag', 'Mittwoch', 'Donnerstag', 'Freitag', 'Samstag'],
+            shortMonths: ['Jan', 'Feb', 'Mrz', 'Apr', 'Mai', 'Jun', 'Jul', 'Aug', 'Sep', 'Okt', 'Nov', 'Dez'],
+            exportButtonTitle: "Exportieren",
+            printButtonTitle: "Drucken",
+            printChart: 'Diagramm ausdrucken',
+            viewFullscreen: 'Im Vollbild anschauen',
+            rangeSelectorFrom: "Von",
+            rangeSelectorTo: "Bis",
+            rangeSelectorZoom: "Zeitraum",
+            downloadPNG: 'Download als PNG-Bild',
+            downloadJPEG: 'Download als JPEG-Bild',
+            downloadPDF: 'Download als PDF-Dokument',
+            downloadSVG: 'Download als SVG-Bild',
+            resetZoom: "Zoom zurücksetzen",
+            resetZoomTitle: "Zoom zurücksetzen",
+            title: 'AAPL Aktienkurs'
+        }
+    }
```

Comment:
There are also accessibility texts that should be translated. Look under [`lang.accessibility`](https://api.highcharts.com/highstock/lang.accessibility). Only the relevant ones need to be changed (shouldn't be that many, but at least look over the things related to the range selector and the screen reader information regions).